src/agent/src/device.rs

```diff
@@ -51,6 +51,7 @@ pub const DRIVER_VFIO_PCI_TYPE: &str = "vfio-pci";
 pub const DRIVER_VFIO_AP_TYPE: &str = "vfio-ap";
 pub const DRIVER_OVERLAYFS_TYPE: &str = "overlayfs";
 pub const FS_TYPE_HUGETLB: &str = "hugetlbfs";
+const NET: &str = "net";
```

Comment:
It needs move this L54 to the place following the L35 BLOCK subsystem. And it will looks better if some comment about it. I don't know what's the meaning of the `NET` in first sight.

Reply:
will do